tools/include/pcid.h

```diff
@@ -155,6 +155,34 @@
 #define PCID_CMD_IS_ASSIGNED            "is_device_assigned"
 #define PCID_MSG_FIELD_RESULT           "result"
 
+/*
+ *******************************************************************************
+ * Get device resources
+ *
+ * This command returns resource list of device
+ *
+ * Request (see other mandatory fields above):
+ *  - "cmd" field of the request must be set to "is_device_assigned".
+ *  - "sbdf" SBDF of the device in format defined by PCID_SBDF_FMT.
+ *  - "permissive" pci->permissive field
```

Comment:
What it does exactly? Imagine that someone tries to implement protocol based on this documentation.

Reply:
Corrected